src/phylum/ci/cli.py

```diff
@@ -91,24 +90,24 @@ def ensure_project(ci_env: CIBase) -> None:
     elif ret.returncode == 14:
         print(f" [-] Project {ci_env.phylum_project} already exists. Continuing with it ...")
     else:
-        print(f" [!] There was a problem creating the project with command: {cmd}")
+        print(f" [!] There was a problem creating the project with command: {' '.join(cmd)}")
         ret.check_returncode()
 
 
 def get_phylum_analysis(ci_env: CIBase) -> dict:
     """Analyze a project lockfile from a given CI environment with the phylum CLI and return the analysis."""
     # Build up the analyze command based on the provided inputs
-    cmd = f"{ci_env.cli_path} analyze -l {ci_env.phylum_label}"
+    cmd = [str(ci_env.cli_path), "analyze", "-l", ci_env.phylum_label]
     if ci_env.phylum_project:
-        cmd = f"{cmd} --project {ci_env.phylum_project}"
+        cmd.extend(["--project", ci_env.phylum_project])
         # A group can not be specified without a project
         if ci_env.phylum_group:
-            cmd = f"{cmd} --group {ci_env.phylum_group}"
-    cmd = f"{cmd} --verbose --json {ci_env.lockfile}"
+            cmd.extend(["--group", ci_env.phylum_group])
+    cmd.extend(["--verbose", "--json", str(ci_env.lockfile)])
 
-    print(f" [*] Performing analysis with command: {cmd} ...")
+    print(f" [*] Performing analysis with command: {' '.join(cmd)} ...")
```

Comment:
🤔 I'm wondering if this should actually be `shlex.join(cmd)` in case a user tries to copy/past this command into a shell...

But that's probably not anything that we need to support and they would already have to remove the `...` at the end. So I guess this is okay as is.

Reply:
> Good idea...but that method was added in Python 3.8

Sad... But luckily, it's a [one line function](https://github.com/python/cpython/blob/0f156c1c5642d3102317e6b4c315098226e6e3ff/Lib/shlex.py#L316-L318) and you don't really need it.

```suggestion
    print(f" [*] Performing analysis with command: {' '.join(shlex.quote(arg) for arg in cmd)} ...")
```

Reply:
Nice find...changed.